From 2c2a9d63d07102e54608ad7691072ebbed0f8be8 Mon Sep 17 00:00:00 2001
From: Mariusz Tkaczyk <mariusz.tkaczyk@linux.intel.com>
Date: Mon, 23 Sep 2024 11:12:53 +0200
Subject: [PATCH 184/201] Remove INSTALL and dev/null

INSTALL is not needed because it added to README.md
dev/null was created accidentally.

Remove them.

Signed-off-by: Mariusz Tkaczyk <mariusz.tkaczyk@linux.intel.com>
---
 INSTALL  | 13 -------------
 dev/null |  0
 2 files changed, 13 deletions(-)
 delete mode 100644 INSTALL
 delete mode 100644 dev/null

diff --git a/INSTALL b/INSTALL
deleted file mode 100644
index f7bcc3e6..00000000
--- a/INSTALL
+++ /dev/null
@@ -1,13 +0,0 @@
-
-To build mdadm, simply run:
-
-   make
-
-to install, run
-
-   make install
-
-as root.
-
-
-No configuration is necessary.
diff --git a/dev/null b/dev/null
deleted file mode 100644
index e69de29b..00000000
-- 
2.41.0

